.gitignore

```diff
@@ -33,6 +33,7 @@ proguard/
 captures/
 
 # IntelliJ
+.idea/
```

Comment:
It's good to ignore the local IntelliJ folder as it mostly contains your local workspace settings.

Reply:
I always try to, I think its because of the multiple git clients I use.

Reply:
I understand. I have had similar issues before. So I completely switched to command line. I do use version control system inside the Android studio for comparison.